### ISSUE TITLE
Add more options to Mouse Button Remap

### ISSUE THREAD

**Issue by omata** [rxhanson/Multitouch-Community#157]
Hello,

Can you add keyboard key combinations with mouse buttons to the Mouse Button Remap? Where not just plain buttons can be remaped, but combine buttons with keyboard keys. As an Example I want to configure buttons 4 and 5 for back and forward in the browser, but I want too combine these with Command or Control Key It to cycle Mission Control Desktops.

BTW: When I try to remap a button of the mouse with a keyboard shortcut, I can't do ^<- or ^-> 

Best regards.

**Comment by rxhanson** [comment]
Agreed, I've been meaning to carry over the "restrictions" section from the gestures (modifiers & applications) into the mouse buttons but haven't gotten to it. 

I'll add the long clicks & repeated clicks to my backlog as well.

> BTW: When I try to remap a button of the mouse with a keyboard shortcut, I can't do ^<- or ^->

I think this might be because macOS has those mapped to switching spaces, and the shortcut recorder doesn't catch them because it isn't listening to keystrokes at a lower level. I don't foresee modifying the shortcut recorder to fix this in the near term since it's fairly involved. 

One option is to unmap those in macOS, configure Multitouch, then remap them.

Another option is to modify Multitouch defaults values without using the UI, but that's more work.
